clang/www/c_status.html

```diff
@@ -501,7 +501,13 @@ <h2 id="c11">C11 implementation status</h2>
     <tr>
       <td>Wide function returns (alternate proposal)</td>
       <td><a href="https://www.open-std.org/jtc1/sc22/wg14/www/docs/n1396.htm">N1396</a></td>
-      <td class="unknown" align="center">Unknown</td>
+      <td class="full" align="center">
+        <details><summary>Yes*</summary>
+        Clang conforms to this paper on all targets except 32-bit x86 without
+        SSE2. However, Clang does not claim conformance or intend to conform to
+        Annex F on that target, so we vacuously conform.
```

Comment:
The "or" IMO breaks the link to "does not" (spoken language is much more fungible with operator precedence...), so I'd formulate this as `not X and not Y`, rather than `not (X or Y)`. Also, I wouldn't call "support on all but one niche target" to be "vacuous".
```suggestion
        SSE2. However, Clang does not claim conformance (nor intends to conform)
        to Annex F on that target, so overall we can still claim full conformance.
```

Reply:
Good call on reformulation as `not X and not Y`, I'll take care of that. I'll also try to reword the "vacuously conform" to be more clear. The intent is: because the changes in the paper only apply to Annex F, and Annex F support is optional, and we don't claim to conform to Annex F on any target (and we will never claim to conform to Annex F for that particular target), we conform to the paper by doing nothing.